index.md

```diff
@@ -1,5 +1,9 @@
 ---
 layout: splash
+<<<<<<< HEAD
+=======
+permalink: /index.html
+>>>>>>> 9b3c08d (#26 | Prototype page added for deployment)
```

Comment:
This is from a git conflict, the conflict must be solved and remove the extra lines.